### PR TITLE
create navbar html, css

### PR DESCRIPTION
**Pull request by SundusAhmed**
relate #3

### CODE REVIEW

**Review by SaraDahman — APPROVED** [review]

**Review by mahmoudhammad309 — DISMISSED** [review]
well done

**Review by MohammedAbuSamra — DISMISSED** [review]
شابوووو خالتو

**Review by sara219 — CHANGES_REQUESTED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
[the remark this answers is not in the document]
Comment (SaraDahman) [reply .vscode/settings.json]: I removed the file

File: index.html [hunk 2]
```diff
@@ -1,13 +1,53 @@
...
+    <div class="container">
+      <!-- this section is containd title, description, button and icons -->
+      <header class="header">
+        <nav>
+          <a href="" class="logo">
+            <img
+              src="/images/logo.svg"
+              alt="logo"
+              width="40px"
+              class="logo-img"
```
[the remark this answers is not in the document]
Comment (SaraDahman) [reply index.html:32]: I added the logo image

**Review by sara219 — APPROVED** [review]

**Review by MohammedAbuSamra — APPROVED** [review]

**Review by SaraDahman — APPROVED** [review]

**Pull request merged**